src/components/accessories/patientSummary/patientSummaryByType/PatientSummaryByType.tsx

```diff
@@ -1,94 +1,125 @@
-  { date: "21/12/2018", type: "Pharmacologic treatment", result: "Result (1)", note: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum." },
-  { date: "21/12/2019", type: "Pharmacologic treatment", result: "Result (1)", note: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum." },
-  { date: "21/12/2019", type: "Pharmacologic treatment", result: "Result (1)", note: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum." },
-  { date: "21/12/2019", type: "Pharmacologic treatment", result: "Result (1)", note: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum." }
-];
+import { IDispatchProps, IStateProps, SummaryType, TProps } from "../types";
 
 const header = ["date", "type", "result"];
 const order = ["date"];
 const label = {
-  "date": "Date",
-  "type": "Typology",
-  "result": "Result",
-  "note": "Additional notes"
-}
+  date: "Date",
+  type: "Typology",
+  result: "Result",
+  note: "Additional notes",
+};
 
-const PatientSummaryByType: FunctionComponent = () => {
+const PatientSummaryByType: FunctionComponent<TProps> = ({
+  hasSucceeded,
+  loadSummaryData,
+  isLoading,
+}) => {
+  const { t } = useTranslation();
+  const [summaryData, setSummaryData] = useState(Array<SummaryData>());
+  const summary = useSelector<IState, ISummaryState>((state) => state.summary);
+  const patient = useSelector((state: IState) => state.patients);
```

Comment:
The same comment in PatientSummaryByDate goes here. We need to avoid this overhead.

Reply:
OK, I applied it. Thanks